canonical-web-and-design/charmhub.io
# Integrate canonicalwebteam.candid module

## Done

- Integrate canonicalwebteam.candid module

## How to QA
- Go to http://localhost:8045/login
- Check that you can login with your Canonical SSO account
- Go to http://localhost:8045/logout

## Issue / Card
Fixes https://github.com/canonical-web-and-design/snap-squad/issues/1833
